integration_tests/sdk/delete_workflow_test.py

```diff
@@ -136,17 +138,23 @@ def test_delete_workflow_saved_objects_twice(client):
         flow_2_id = flow_list[1]
 
         tables = client.flow(flow_1_id).list_saved_objects()
-        assert "delete_table" in [item.object_name for item in tables[get_integration_name()]]
+        tables_1 = set([item.object_name for item in tables[get_integration_name()]])
+        assert "delete_table" in tables_1
 
         tables = client.flow(flow_2_id).list_saved_objects()
-        assert "delete_table" in [item.object_name for item in tables[get_integration_name()]]
+        tables_2 = set([item.object_name for item in tables[get_integration_name()]])
+        assert "delete_table" in tables_2
+
+        assert tables_1 == tables_2
 
         # No SDK function to do this so we query the endpoint directly to see delete_table is properly created at the integration.
```

Comment:
Why can't we do `integration.sql("select * from delete_table").get()` to check that the table exists? I'd prefer to keep actual endpoints out of the SDK if possible.

Reply:
That's a good idea! Thanks for the suggestion. I will just be checking the following:
```
        with pytest.raises(AqueductError) as e_info:
             integration.sql("SELECT * FROM delete_table").get()
```

Let me know if there is anything else you think would be good to check.